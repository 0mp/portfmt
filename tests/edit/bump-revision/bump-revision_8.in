PORTNAME=	lepton-eda
PORTVERSION=	1.9.7
PORTREVISION?=	2
DISTVERSIONSUFFIX=	-20181211
CATEGORIES=	cad
