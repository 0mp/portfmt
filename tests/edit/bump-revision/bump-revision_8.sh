${PORTEDIT} bump-revision bump-revision_8.in | \
	diff -L bump-revision_8.expected -L bump-revision_8.actual \
		-u bump-revision_8.expected -
