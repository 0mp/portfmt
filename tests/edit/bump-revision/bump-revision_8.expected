PORTNAME=	lepton-eda
PORTVERSION=	1.9.7
DISTVERSIONSUFFIX=	-20181211
PORTREVISION?=	3
CATEGORIES=	cad
